projects/storefrontlib/src/cms-pages/product-listing-page/product-listing-page.module.ts

```diff
@@ -0,0 +1,43 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import {
+  CmsPageGuard,
+  PageLayoutComponent,
+  suffixUrlMatcher,
+} from '../../cms-structure/index';
+@NgModule({
+  imports: [
+    RouterModule.forChild([
+      {
+        path: null,
+        canActivate: [CmsPageGuard],
+        component: PageLayoutComponent,
+        data: { cxRoute: 'category' },
+      },
+      {
+        matcher: suffixUrlMatcher,
```

Comment:
Can we move the suffix route to the end (after the brand route)? Reason:
The default brand route is `'Brands/:brandName/c/:brandCode'`, but suffix route matches everything ending with `/c/...`, so now we won't get proper params in the PLP component. For example for `Brands/Sony/c/123` instead of:
`{ brandName: 'Sony', brandCode: '123' }`
we get:
`{ param0: 'Brands', param1: 'Sony', categoryCode: '123' }`.